src/pymail

```diff
@@ -84,72 +103,91 @@ def email_recv(use_sys_argv):
-    # Get the latest email
-    result, data = mail.fetch(email_id, "(RFC822)")
-    raw_email = data[0][1]
 
-    print(raw_email)
+def get_email_pass():
+    """Get the email and password from the user.
+    Turn off output to terminal for password.
+    Returns a tuple with email and password.
+    """
+    
+    _email = raw_input("Email address = ")
+    print("Email password = ")
+    # Turn off output in terminal
+    os.popen("stty -echo")
+    # Get the password which whill not be shown
+    _pass = raw_input()
+    # Turn on output in terminal
+    os.popen("stty echo")
+
+    print("\n")
+
+    return (_email, _pass)
+
 
 def login():
-    """Get account credentials"""
+    """Get account credentials from files or from user
+    input if the file specified does not exist.
+    """
+
+    # Define global variables
     global email_addr
     global email_pass
-    if os.path.isfile(install_path + "mailaddr") == True:
-        mail_addr_file = open(install_path + "mailaddr", 'r')
-        email_addr = mail_addr_file.read()
-        mail_pass_file = open(install_path + "mailpass", 'r')
-        email_pass = mail_pass_file.read()
+
+    # If the file specified exists
+    if os.path.isfile(install_path + "mailaddr"):
+    	# Get the variables from the two files; install_path + mail(addr/pass)
+    	email_addr, email_pass = [open(install_path + 'mail%s' % x).read().close() for x in ['addr', 'pass']]
     else:
-        email_addr = raw_input("Email address = ")
-        print("Email password = "),
-        os.popen("stty -echo")
-        email_pass = raw_input()
-        os.popen("stty echo")
-        print("\n")
+        email_addr, email_pass = get_email_pass()
+
 
 def config():
-    """Get and save account credentials"""
+    """Get the credentials from the user and store them
+    in install_path + mail(addr/pass).
+    """
+
+    # TODO: Implement encryption.
     print("BE AWARE THAT YOUR CREDENTIALS WILL NOT BE ENCRYPTED.")
-    email_addr = raw_input("Email address = ")
 
-    print("Email password = "),
-    os.popen("stty -echo")
-    email_pass = raw_input()
-    os.popen("stty echo")
-    print("\n")
+    email_addr, email_pass = get_email_pass()
 
-    mail_addr_file = open("mailaddr", 'w')
-    mail_pass_file = open("mailpass", 'w')
+    # List comprehensions are awesome
+    # Pretty self-explanatory, but saves the email and pass to the files
+    [open(install_path + 'mail%s' % x, 'w').write(y).close() for x, y in [['addr', email_addr], ['pass', email_pass]]]
 
-    mail_addr_file.write(email_addr)
-    mail_pass_file.write(email_pass)
 
 def pymail_help():
-    """Prints basic help"""
-    print("Usage:")
-    print("pymail help")
-    print("pymail config")
-    print("pymail send")
-    print("pymail send <receiver> <subject> <body>")
-    print("pymail receive")
-    print("pymail receive <n>")
-    print("pymail file <file> <delete_concept (y/n)>")
+    """Iterates over a list of options and prints them to the screen.
```

Comment:
Read it on the interwebz, must be true. Doesn't work at least, btw are you coding in Python 3.x?